### PR TITLE
Meteor package

### PR DESCRIPTION
**Pull request by rgnevashev**

### CODE REVIEW

File: .versions [hunk 1]
```diff
@@ -0,0 +1,32 @@
+babel-compiler@5.8.24_1
```
Comment (arshaw) [inline .versions]: could we have fullcalendar's build process handle generating this? similar to how the dist files are generated for a release, but not committed to master.

also, what are all these packages? are they dependencies of fullcalendar? i hope not.

File: bower.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fullcalendar",
   "title": "FullCalendar",
-  "version": "0.0.0",
+  "version": "2.6.1",
```
Comment (arshaw) [inline bower.json]: this should be taken care of by the build process

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fullcalendar",
   "title": "FullCalendar",
-  "version": "0.0.0",
```
Comment (arshaw) [inline package.json]: this should be taken care of by the build process (that's currently how the version bump works)

File: package.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Package.describe({
```
Comment (arshaw) [inline package.js]: im confused by the this file. i thought Dojo owned package.js no? fwiw, here's moment's:
https://github.com/moment/moment/blob/develop/package.js

**Comment by arshaw** [comment]
could you please give more info on what the Meteor package manager is and why FullCalendar should hop onboard?

**Comment by rgnevashev** [comment]
It is meteor package, for atmospherejs...
I think you could be know about meteor...

**Comment by guidouil** [comment]
now meteor is moving it's package management to npm, maybe you should to...

**Comment by arshaw** [comment]
fullcalendar is already installable as an npm module
https://github.com/fullcalendar/fullcalendar/blob/master/package.json
`npm install fullcalendar`

glad to hear meteor will rely on npm. in that case, i'll close this ticket out

**Pull request closed without merging**